tmt/steps/prepare/install.py

```diff
@@ -616,7 +616,7 @@ def go(
 
         else:
             raise tmt.utils.PrepareError(
-                f'Package manager "{guest.facts.package_manager}" is not supported.')
+                'No supported package manager found.')
```

Comment:
Since you're in the area... may we get two different messages? One can be more specific, and it was the one you hit today, i.e. package manager was supported by tmt, but unsupported by `prepare/install`. How about this?

```python
elif guest.facts.package_manager is None:
    raise tmt.utils.PrepareError('Unrecognized package manager.')

else:
    raise tmt.utils.PrepareError(f"Package manager '{guest.facts.package_manager}' is not supported by 'prepare/install'.")
```

Reply:
Changed in 9f8f00de8d56b55fbbbd87b91b9841a18f10081b

Reply:
ty